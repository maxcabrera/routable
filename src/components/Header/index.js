import React from 'react'
import styled from 'styled-components'

const HeaderWrapper = styled.section`
  padding: 1em;
  background: rgba(60,74,136,1);
  text-align: center;
  
  small,
  h1 {
    color: #fafbff;
  }
  
  h1 {
    padding: 0;
    margin: 0;
  }
`;

const Header = (props) => {
  const { title } = props
  return (
    <HeaderWrapper>
      <h1>{title}</h1>
      <small>by Max Cabrera</small>
    </HeaderWrapper>
  )
}

export default Header