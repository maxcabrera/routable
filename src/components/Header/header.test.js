import Header from './'

describe('Header section', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<Header title="Issues Management System" />)
  })

  test('Snapshot test for Header', () => {
    const tree = renderer.create(<Header title="Issues Management System" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})