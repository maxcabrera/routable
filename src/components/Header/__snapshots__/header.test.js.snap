// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header section Renders without crashing 1`] = `
<section
  className="sc-bdVaJa kOycRs"
>
  <h1>
    Issues Management System
  </h1>
  <small>
    by Max Cabrera
  </small>
</section>
`;

exports[`Header section Snapshot test for Header 1`] = `
<section
  className="sc-bdVaJa kOycRs"
>
  <h1>
    Issues Management System
  </h1>
  <small>
    by Max Cabrera
  </small>
</section>
`;
